AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests/ResolverTests/AuthorizationState/FetchAuthSessionStateResolverTests.swift

```diff
@@ -30,7 +31,7 @@ class FetchAuthSessionStateResolverTests: XCTestCase {
 //        let cognitoSession = AWSAuthCognitoSession.testData
 
         let validSequences: [FetchAuthSessionStateSequence] = [
-//            StateSequence(oldState: .initializingFetchAuthSession,
+            //            StateSequence(oldState: .initializingFetchAuthSession,
```

Comment:
seems like an unintentional change?

Reply:
looks to be attempting to resolve alignment, I've removed the commented out code.